include/alpaka/vec/Vec.hpp

```diff
@@ -511,8 +512,7 @@ namespace alpaka
         }
 
     private:
-        // 16 Byte alignment for usage inside of CUDA kernels.
-        alignas(16u) TSize m_data[TDim::value];
+        alignas(ALPAKA_OPTIMAL_ALIGNMENT_SIZE(TSize[TDim::value])) TSize m_data[TDim::value];
```

Comment:
The macro name must be `ALPAKA_OPTIMAL_ALIGNMENT` sry that was my fault in my first annotation.